src/main/cpp/teleopcontrol/TeleopControlMap.h

```diff
@@ -158,7 +158,7 @@ robin_hood::unordered_map<TeleopControlFunctions::FUNCTION, const TeleopControlB
     // {TeleopControlFunctions::ALIGN_FLOOR_GAME_PIECE, driverXButton},
 
     {TeleopControlFunctions::RESET_POSITION, driverLStickPressed},
-    {TeleopControlFunctions::ROBOT_ORIENTED_DRIVE, driverRStickPressed},
+    {TeleopControlFunctions::TIPCORRECTION_TOGGLE, driverRStickPressed},
```

Comment:
Why is getting a chassis bot up and running changing the button mapping for the driver on the compbot?

Reply:
I can change it back, but in main right now we are assigning tip correction and robot oriented drive to the same button.